code/assets/css/header.css

```diff
@@ -0,0 +1,61 @@
+.main-header {
+  background-color: var(--secondaryColorNormal);
+}
+
+.header-container {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: calc(var(--spacing) * 4) 0;
+}
+
+.logo img {
+  width: 90%;
+}
+
+.hamburger-menu {
+  color: var(--primaryColorDarker);
+  font-size: 2em;
+}
+
+#main-menu-cnt {
```

Comment:
We shouldn't style by id's - let's add class to that HTML element and style by it